content/updates/2020-09/1-flat-files-exports-with-labels.md

```diff
@@ -0,0 +1,30 @@
+---
+pim_announcement_img: ./img/Product-export-profile.png
+pim_announcement_alt_img: "Flat files exports with labels"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Export flat files with labels
+::: meta-data type="New" features="Collaboration, Productivity" available="mid-September" link-to-doc="../articles/exports.html#update-an-export-profile"
+:::
+
+To save you time, it is now possible to export your product data in flat files with labels and without codes.  
+This improvement has been added to both existing features: `Quick export` and `Product export profiles`.  
+You will notice an optional button named `With label`, helping you to export your product data with labels.  
+When activated, the **headers** (properties & attributes) and the **values** in your exported flat file will be displayed in the locale you will choose instead of displaying their codes.
+
+For instance, if you have a multi-select attribute named `TS_STYLE032`, it will be exported as a label, `Tee-shirt style 032`.
+Same thing with the values of this attribute `T_NECK` and `L_SLEEVES` would be displayed a labels `Turtleneck` and `Long sleeves`.
+
+Please note that this improvement is optional.
+
+If you choose to export with labels, you won't be able to import your file back into your PIM.
```

Comment:
I'd remove this, it's not very relevant for the news, it's more for the Helpcenter article.
```suggestion
```